### ISSUE TITLE
Ability to reply to email notifications

### ISSUE THREAD

**Issue by t3chguy** [matrix-org/synapse#12758]
Moved from https://github.com/vector-im/element-web/issues/1572

> Two ways of doing this: an email->matrix bridge, or 'reply' links on the notifs. We already have the latter, so I assume this is the former. The bridge would not be easy though as if you have multiple notifs in the email, which are you responding to?

> A better solution would be a separate email<->matrix bridge I suspect.

**Comment by babolivier** [comment]
This looks to me like a request for an email -> matrix bridge, which isn't relevant for Synapse.

**Comment by t3chguy** [comment]
@babolivier no it is a request to be able to reply to the notification emails sent by Synapse, the suggestion for a bridge is one of how to power it

**Comment by babolivier** [comment]
Right. Though there's no work planned to add an SMTP server to Synapse, and we are unlikely to accept a PR for it due to the complexity it brings (especially given it's not a very often requested feature).

**Comment by ara4n** [comment]
Whether or not we support the ability to reply to email notification should be decided in the context of making the best user-facing product.  For instance, Google let you do it for their products - and if we want to support the same for Matrix clients, we should.  How that's implemented (SMTP servers in Synapse or some seperate API you can hook into) is not the reason to reject it as a feature proposal.

**Comment by richvdh** [comment]
I don't think it's appropriate to track this in the Synapse project - it's not really a bug in synapse and as @babolivier says, it's unlikely something we'd add. Moving to matrix-spec since it's a generic "make matrix better, somehow" feature.
